doc/mgr/dashboard.rst

```diff
@@ -333,6 +333,21 @@ commands::
 
   $ ceph dashboard ac-user-create <username> <password> administrator
 
+Enable a Locked User
+^^^^^^^^^^^^^^^^^^^^
+
+If a user account is disabled as a result of multiple invalid login attempts, then
+it needs to be manually enabled by the administrator. This can be done by the following
+command::
+
+  $ ceph dashboard ac-user-enable <username>
+
+In addition to this, the user can get and set the default number of lockout attempts using
+these commands respectively::
+  
+  $ ceph dashboard get-account-lockout-attempts
+  $ ceph dashboard set-account-lockout-attempts <value:int>
+
```

Comment:
I'd put this in it own chapter (`Account lock-out`) and would begin with:
* What the account lock-out is and what for (brute-force/dictionary attacks)
* That it's enabled by default
* How to configure (including
* How to disable it (with a warning no to do)

Reply:
Done. Thanks.